handbook/company/pricing-features-table.yml

```
@@ -123,6 +123,9 @@
   demos:
     - description: Security engineers at a top gaming company wanted to get demographics off their macOS, Windows, and Linux machines about who the user is and who's logged in.
       moreInfoUrl: https://docs.google.com/document/d/1qFYtMoKh3zyERLhbErJOEOo2me6Bc7KOOkjKn482Sqc/edit
+    - description: Data engineers at a top biotech corporation has a need for knowing who is logged into their devices. 
+      quote: So we don't know exactly what's going on after we deploy the device, we know that they are compliant with the security because we are running these stuff, but we don't know certainly who is running, who is logging in the device?
```

```suggestion
      quote: So we don't know exactly what's going on after we deploy the device, we know that they are compliant with Security's baselines because we are running this stuff, but we don't know with certainty who is logging into the device.
```
What does this mean? (took a shot at edits above)

---

@mikermcneil because they enforce the baselines, they know which devices have them installed and which ones do not. But even for the ones that do, they have no way of guaranteeing only those assigned the devices are using them. They are getting visibility that devices are online and meeting the baselines deployed or checked against, and that seems to be it. A compliant device in the hands of the assigned end user looks the same to them as a compliant device in the hands of someone else (or potentially even someone not at the company).